src/module/documents/active-effect.mjs

```diff
@@ -1,4 +1,15 @@
 export class DrawSteelActiveEffect extends ActiveEffect {
+
+  /**
+   * Automatically deactivate effects with expired durations
+   */
+  get isSuppressed() {
+    if (Number.isNumeric(this.duration?.remaining)) {
```

Comment:
Can this be simplified to just `if (this.isTemporary && !this.duration.remaining) return true;` ?

Reply:
Ah I immediately realize why not (effects with just `statuses`).